src/components/tiles/map-tile/map.tsx

```diff
@@ -6,7 +6,12 @@ type MapProps = {
 
 export function Map({ active }: MapProps) {
   return (
-    <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 436.39 700.05' className='md:max-h-[800px] max-h-max'>
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      viewBox='0 0 436.39 700.05'
+      height={800}
+      className='mdx:max-h-[800px] max-h-[400px] max-w-full'
+    >
```

Comment:
The max-h-[400px] is weird for sm and md screens, on the outer div this is set to 1000px